client/scss/layouts/_home.scss

```
@@ -46,22 +46,23 @@
     li svg.icon {
       width: 1em;
       height: 1em;
-      margin-inline-end: 0.5em;
+      margin-inline-end: 0.15em;
     }
 
     a {
       position: relative;
       display: block;
       width: 100%;
       height: 100%;
+      color: theme('colors.secondary.DEFAULT');
```

```suggestion
      color: currentColor:
```

As discussed the other day, best to use the currentColor approach Here if possible as it avoids an additional colour setting and makes it easier to override the parent container styles and not have to also apply it to the link. 

Alternatively you could use `color: inherit;` to achieve the same goal.